libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -122,8 +123,8 @@ void AP_InertialSensor_Backend::_notify_new_gyro_raw_sample(uint8_t instance,
         uint64_t now = AP_HAL::micros64();
         struct log_GYRO pkt = {
             LOG_PACKET_HEADER_INIT((uint8_t)(LOG_GYR1_MSG+instance)),
-            time_us   : now,
-            sample_us : sample_us?sample_us:now,
+            time_us   : sample_us?sample_us:now,
```

Comment:
I haven't looked at all the drivers to confirm or deny that. But if that is the case then we should be consistent in all drivers: either use time gathered when constructing the log message or use the time collected in the driver. If you don't want to do that, then I think that only having one field is wrong; you wouldn't be able to look at a log and know when time_us was collected without going to see what sensor/driver it belonged to.